src/apps/favorites-list/favorites-list.test.ts

```diff
@@ -45,31 +51,75 @@ describe("Favorites list", () => {
     cy.visit("/iframe.html?id=apps-favorite-list--primary&viewMode=story");
   });
 
+  // Test that the list of favorites is displayed with materials correctly
   it("Favorites list basics", () => {
```

Comment:
These testings were built by ITK before they yeeted out of the project. 
The way we do it in the other tests (written by Reload) is that we make many small tests so that we can see in the Git log when a test doesn't pass - which one it was. 
I would really appreciate it if we divided the test into several many tests to that every it() tests one main thing. So when things fail, we see it in the log that for example "it("shows the header") test failed, or it("material links to correct material") test failed. It's so much easier when things aren't passing that way. It happened to me way too many times in the past that one of these monster tests didn't pass and then I had to actually run it locally to figure out what's wrong and look for it inside the it() - which part it is that is failing. 
Another downside with this approach (many test cases inside one it() ) is that the test doesn't run to the end if something fails in the beginning. If it's divided into many subtests we can precisely see which parts work and which don't. Even if it's the first test that fails - all the others will still run so we can see if those are failing too. 
So my recommendation is to take every // Comment ..and make these into separate it() cases. 🌻

Reply:
While I totally agree with your point, and that we generally should divide tests into something small and contained, I don't think that it make sense with this exact case. 

I have changed the wording of the test, as it was too vague, to actually tell what it did, and when or if it fails, we know its because the favorite list is not rendered as expected.